AdblockPlusSafari/AdblockPlusExtras.m

```diff
@@ -183,28 +184,54 @@ - (void)checkActivatedFlag
   }
 }
 
-- (void)updateFilterLists:(BOOL)userTriggered
+- (void)updateAllFilterLists:(BOOL)userTriggered
 {
-  NSMutableDictionary *filterLists = [self.filterLists mutableCopy];
-  for (NSString *filterListName in self.filterLists) {
+  [self updateFilterLists:self.filterLists userTriggered:YES];
+}
+
+- (void)updateFilterLists:(NSDictionary<NSString *, NSDictionary<NSString *, id> *> *)filterLists userTriggered:(BOOL)userTriggered
+{
+  NSMutableDictionary *mutableFilterLists = [self.filterLists mutableCopy];
```

Comment:
How I understand it, `self.filterLists` is mutable as well. So how about renaming `mutableFilterLists` to `updateFilterLists` or something in that vein?